catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java

```diff
@@ -348,6 +269,111 @@ public void addFields(Metacard metacard, SolrInputDocument solrInputDocument)
     solrInputDocument.addField(SchemaFields.METACARD_TYPE_OBJECT_FIELD_NAME, metacardTypeBytes);
   }
 
+  private void addAttributeValues(Metacard metacard, SolrInputDocument solrInputDocument)
+          throws MetacardCreationException {
+    MetacardType schema = metacard.getMetacardType();
+
+    for (AttributeDescriptor ad : schema.getAttributeDescriptors()) {
+      if (hasCorrespondingAttributeWithValues(metacard, ad)) {
+        List<Serializable> attributeValues = metacard.getAttribute(ad.getName()).getValues();
+          AttributeFormat format = ad.getType().getAttributeFormat();
```

Comment:
Formatting looks off here.

Reply:
I formatted everything on the last commit 😄